DevoxxClientMobile/src/main/resources/com/devoxx/common.css

```diff
@@ -490,6 +490,16 @@
     -fx-padding: 0.910488em;
 }
 
+.badge-scanner.fab {
+    -fx-graphic: url(images/qrcode-scan.png);
```

Comment:
It looks good so far, but it probably would be better an SVG path. If this is not possible, we can use this image.

Reply:
The button in FAB is not exposed making it difficult to add a node as graphic to the button.